package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "eos-icons-vscode",
+  "version": "0.0.1",
+  "publisher": "EOS-uiux-Solutions",
+  "license": "MIT",
+  "description": "The iconic and open source font made for EOS Design System for VSCode.",
+  "keywords": ["iconic", "font", "eos", "icons", "product-icon"],
+  "author": "SUSE UX/UI team",
+  "engines": {
+    "vscode": ">=1.53.0"
+  },
+  "scripts": {
+    "cleandeps": "rm -rf node_modules .env3",
+    "package": "vsce package --yarn",
+    "prepare": "python3 -m venv .env3 && . .env3/bin/activate && pip install fonttools",
+    "vscode:prepublish": ". .env3/bin/activate && ./create_icon_theme.py"
```

Comment:
when I run the prepublisher I get the following error:

![image](https://user-images.githubusercontent.com/5013096/109549130-d37cfc00-7acd-11eb-8d11-44e9c2e92ff7.png)

Reply:
That should not happen if you run `yarn run prepare` beforehands.